src/Cache/CacheClient.php

```diff
@@ -38,7 +37,7 @@
 use Psr\Log\LoggerAwareInterface;
```

Comment:
any idea what all the qodana stuff below is?  i feel like we should either disable that or add whatever it is doing to our CI linting?  doesn't have to be in this PR but it does make it kinda hard to review :)

Reply:
No idea, but these are clearly false positives. I downloaded the binary and it's definitely going to take some digging to see what's up with this thing. I'll go ahead and disable it for now until I have some time to take a closer look.